CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # v0.10.0 [UNRELEASED]
 
 ## Features
-* Added the --chat-device flag to override a device used to run chat model.
-
-# v0.9.2 [UNRELEASED]
+* Introduced the `--chat-device` flag to specify the device used to run the chat model.
```

Comment:
Any reason we don't reuse the `--device` flag for this?

Reply:
https://github.com/TabbyML/tabby/pull/1685